python/py-xarray/Portfile

```diff
@@ -4,9 +4,8 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-xarray
-version             2022.10.0
+version             2022.12.0
 revision            0
-
```

Comment:
no unneeded whitespace changes please; this was perfectly fine and typically there is a newline after the revision.